zenoh/__init__.pyi

```diff
@@ -54,7 +90,7 @@ class Config:
     def from_file(cls, path: str | Path) -> Self: ...
     @classmethod
     def from_json5(cls, obj: Any) -> Self: ...
-    @property
+    property
```

Comment:
Is this a wanted change or some simple error?

Reply:
It's a typo indeed, I'm fixing it